config/config.go

```diff
@@ -210,12 +211,33 @@ func FromViper(v *viper.Viper) (*Config, error) {
 		cfg.Excludes = cfg.Global.Excludes
 	}
 
+	// validate formatter names do not contain invalid characters
+
+	nameRegex := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
I can never remember: does this do the right thing with newlines? In python, ^ and $ sometimes (I think in multiline mode?) mean "beginning of line" and "end of line", which means people can sneak newline characters in. That's what `\A` and `\B` or python's newer "no really, search for a match for the *whole* string, not a part of it" function is for.

Sorry for the python esoterica. No idea if any of this applies here. Maybe just expand the test above to have a name that is actually 2 valid names separated by a newline char. Hopefully this code already does the right thing.

Reply:
Golang doesn't have an explicit multiline mode for regexes; you must explicitly add matches for new lines in your expressions. We aren't doing that.